website/docs/components/link/standalone/partials/code/how-to-use.md

```diff
@@ -1,71 +1,67 @@
 ## How to use this component
 
-The most basic invocation requires both `icon` and `text` to be passed and one of the two `@href` or `@route` arguments.
+The most basic invocation requires both `@icon` and `@text`, and either an `@href` or `@route` argument.
 
 ```handlebars
 <Hds::Link::Standalone @icon="film" @text="Watch tutorial video" @href="..." />
 ```
 
 ### Icon position
 
-By default the icon is placed before the text. If you would like to position it after the text, define `@iconPosition`:
+By default the icon is placed before the text. If you would like to position it after the text, set `@iconPosition` to `trailing`.
 
 ```handlebars
 <Hds::Link::Standalone @icon="film" @iconPosition="trailing" @text="Watch tutorial video" @href="..." />
 ```
 
 ### Color
 
-There are two available colors for a Link (Standalone): `primary` and `secondary`. The default is `primary`. To use a different color, declare another value for `@color`:
+There are two available colors for a Standalone Link: `primary` and `secondary`. The default is `primary`. To use a different color, set `@color` to `secondary`.
```

Comment:
```suggestion
There are two available colors for a Standalone Link: `primary` and `secondary`. The default is `primary`. To use the other option, set `@color` to `secondary`.
```

Reply:
To me, this part kind of makes it sound like there are other options than just two: "To use a different color".

There may be a better way to phrase it than my suggestion however.